champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/brute/axe/ThreateningShout.java

```diff
@@ -9,44 +8,76 @@
 import me.mykindos.betterpvp.champions.champions.skills.data.SkillActions;
 import me.mykindos.betterpvp.champions.champions.skills.types.CooldownSkill;
 import me.mykindos.betterpvp.champions.champions.skills.types.InteractSkill;
+import me.mykindos.betterpvp.core.combat.events.CustomDamageEvent;
 import me.mykindos.betterpvp.core.components.champions.Role;
 import me.mykindos.betterpvp.core.components.champions.SkillType;
 import me.mykindos.betterpvp.core.effects.EffectTypes;
+import me.mykindos.betterpvp.core.framework.updater.UpdateEvent;
+import me.mykindos.betterpvp.core.listener.BPvPListener;
+import me.mykindos.betterpvp.core.utilities.UtilBlock;
+import me.mykindos.betterpvp.core.utilities.UtilDamage;
 import me.mykindos.betterpvp.core.utilities.UtilEntity;
 import me.mykindos.betterpvp.core.utilities.UtilFormat;
 import me.mykindos.betterpvp.core.utilities.UtilMessage;
+import org.bukkit.Bukkit;
+import org.bukkit.Location;
+import org.bukkit.Particle;
 import org.bukkit.Sound;
+import org.bukkit.block.Block;
 import org.bukkit.entity.LivingEntity;
 import org.bukkit.entity.Player;
+import org.bukkit.event.Listener;
 import org.bukkit.event.block.Action;
+import org.bukkit.event.entity.EntityDamageEvent;
+import org.bukkit.scheduler.BukkitRunnable;
+
+import java.util.*;
 
 @Singleton
-public class ThreateningShout extends Skill implements InteractSkill, CooldownSkill {
+@BPvPListener
+public class ThreateningShout extends Skill implements Listener, InteractSkill, CooldownSkill {
 
     private double radius;
     private double baseDuration;
     private double durationIncreasePerLevel;
     private int vulnerabilityStrength;
+    private final Map<Player, List<Location>> playerPointsMap;
+    private final Map<Player, Integer> playerPointIndexMap;
+    private final Map<Player, Set<LivingEntity>> playerAffectedEntitiesMap;
+    private final Map<Player, Set<LivingEntity>> playerDamagedEntitiesMap;
```

Comment:
Feel like if you need this many maps you're probably just better off making an object like ThreateningShoutData